src/engineering.html

```diff
@@ -10,323 +10,449 @@
     <link rel="shortcut icon" type="image/x-icon" href=""> <!--add image icon here-->
     <link href="css/style.css" rel="stylesheet">
     <title>508.dev Engineers</title>
-  </head>
-  <body>
-    <header>
-      <h1 class="desktop">
+</head>
+<body>
+<header>
+    <h1 class="desktop">
         508.dev
-      </h1>
```

Comment:
reissue without these indentation changes, please

Reply:
Updated. Please let me know if any further concerns. 

By the way, I notice there is a div tag missing (no closed &lt;/div&gt;  error). So I reformat the code.

Reply:
oh, was the indentation stuf from that? well, good find if so, can you issue a separate PR about that though?